lightly/models/batchnorm.py

```diff
@@ -63,10 +65,11 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
                 self.eps,
             ).view(N, C, H, W)
         else:
+            assert self.running_mean is not None and self.running_var is not None
```

Comment:
I would avoid having assert statements in time critical places like forward passes. That's why I added the ignore in the first place.